x11-misc/xkblayout-state/xkblayout-state-20171214.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+# Commit Date: 27 Nov 2017
+COMMIT="d023bd7f58a54da08c3aad3c234a629b8a010892"
```

Comment:
I'd suggest naming it EGIT_COMMIT for consistency.

Reply:
Done.